web/templates/shared/call-for-speakers.html.slim

```diff
@@ -0,0 +1,12 @@
+.meetup-speakers
+  h2.heading Call for Speakers
+  p.description The floor is open to anyone who has an Elixir story to share. We would love to hear from you! Just a few notes:
+  ul
+    li No previous speaking experience is required, we welcome first timers.
+    li We encourage beginner-friendly talks.
+    li Talks should be 45 minutes long, including Q&A which should only take 5-8 minutes.
+    li Please take note that the meet up will be broadcasted on Google Hangouts.
+  p.description 
+    | Interested in speaking? We would love to hear from you! Feel free to drop your proposal 
```

Comment:
Also, @samrosecruz please remove the `We would to hear from you!` phrase. Its kinda redundant with [this](https://github.com/Alchemists-PH/Website/pull/19/files#diff-ae18019d95890226a229ba287d31aaa8R3). I also removed that phrase na sa ticketbase. Sorry for that. :D